server/src/main/java/org/candlepin/dto/api/v1/ConsumerContentOverrideDTO.java

```diff
@@ -0,0 +1,155 @@
+        this.populate(source);
+    }
+
+    /**
+     * Fetches the activation key that's overriding the content. If the activation key has not been
+     * set, this method returns null.
+     *
+     * @return
+     *  The activation key overriding the content, or null if the activation key has not been set
+     */
+    @JsonIgnore
+    public ConsumerDTO getConsumer() {
+        return this.consumer;
+    }
+
+    /**
+     * Sets or clears the activation key that's overriding the content. If the provided key is null,
+     * any key currently set will be cleared.
+     *
+     * @param consumer
+     *  The activation key that's overriding the content, or null to clear the key
+     *
+     * @return
+     *  A reference to this DTO
+     */
+    @JsonProperty("key")
+    public ConsumerContentOverrideDTO setConsumer(ConsumerDTO consumer) {
+        this.consumer = consumer;
+        return this;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public String toString() {
+        ConsumerDTO consumer = this.getConsumer();
+
+        return String.format("ConsumerContentOverrideDTO [consumer: %s, content: %s, name: %s, value: %s]",
+            this.getContentLabel(),
+            this.getName(),
+            this.getValue(),
+            (consumer != null ? consumer.getId() : null));
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public boolean equals(Object obj) {
+        if (obj == this) {
+            return true;
+        }
+
+        if (obj instanceof ConsumerContentOverrideDTO && super.equals(obj)) {
+            ConsumerContentOverrideDTO that = (ConsumerContentOverrideDTO) obj;
+
+            ConsumerDTO thisConsumer = this.getConsumer();
+            ConsumerDTO thatConsumer = that.getConsumer();
+
+            String thisConsumerId = thisConsumer != null ? thisConsumer.getId() : null;
+            String thatConsumerId = thatConsumer != null ? thatConsumer.getId() : null;
+
+            return thisConsumerId != null ? thisConsumerId.equals(thatConsumerId) : thatConsumerId == null;
+        }
+
+        return false;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public int hashCode() {
+        ConsumerDTO consumer = this.getConsumer();
+
+        HashCodeBuilder builder = new HashCodeBuilder(37, 7)
+            .append(super.hashCode())
+            .append(consumer != null ? consumer.getId() : null);
+
+        return builder.toHashCode();
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public ConsumerContentOverrideDTO clone() {
+        ConsumerContentOverrideDTO copy = super.clone();
+
+        ConsumerDTO consumer = this.getConsumer();
+        copy.setConsumer(consumer != null ? consumer.clone() : null);
+
+        return copy;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public ConsumerContentOverrideDTO populate(ConsumerContentOverrideDTO source) {
+        super.populate(source);
+
```

Comment:
Should also add `this.setConsumer(source.getConsumer());`

Reply:
Fixed.